Finished relicense (#428)
SlimeVR/SlimeVR-Server: README.md

Latest instructions are [on our site](https://docs.slimevr.dev/server-setup/slimevr-setup.html).

## License Clarification
**SlimeVR software** (including server, firmware, drivers, installer, documents, and others - see
licence for each case specifically) **is distributed under a dual MIT/Apache 2.0 License

> ([LICENSE-MIT] and [LICENSE-APACHE]). These licenses are the copyright of the SlimeVR

Reviewer: This is worded wrong.

```suggestion
([LICENSE-MIT] and [LICENSE-APACHE]). The software is the copyright of the SlimeVR
```

> ([LICENSE-MIT] and [LICENSE-APACHE]). These licenses are the copyright of the SlimeVR

Reviewer: This is worded wrong.

```suggestion
([LICENSE-MIT] and [LICENSE-APACHE]). The software is the copyright of the SlimeVR
```

contributors.** 

**However, these licenses have some limits, and if you wish to distribute software based
on SlimeVR, you need to be aware of them:**

* When distributing any software that uses or is based on SlimeVR, you have to provide
  to the end-user at least one of the original, unmodified [LICENSE-MIT] or

>   [LICENSE-APACHE] files from SlimeVR. This includes the `Copyright (c) 2022 SlimeVR
>   Contributors` part of the license. It is insufficient to use a generic MIT

Author: ```suggestion
  [LICENSE-APACHE] files from SlimeVR. This includes the `Copyright (c) 2020 Eiren Rain
  and SlimeVR Contributors` part of the license. It is insufficient to use a generic MIT
```
